rotkehlchen/db/schema.py

```diff
@@ -328,6 +328,17 @@
 );
 """
 
+DB_CREATE_EVM_TRANSACTIONS_AUTHORIZATION = """
+CREATE TABLE IF NOT EXISTS evm_transactions_authorization (
+    tx_hash BLOB NOT NULL,
+    chain_id INTEGER NOT NULL,
+    nonce INTEGER NOT NULL,
+    delegated_address TEXT NOT NULL,
+    PRIMARY KEY(tx_hash, chain_id, delegated_address),
+    FOREIGN KEY(tx_hash, chain_id) REFERENCES evm_transactions(tx_hash, chain_id) ON DELETE CASCADE
```

Comment:
No no no. I told you it should be referencing the transaction table. Why complicate things with composite primary keys and composite foreign key? Not only is this slower but it takes a lot more space.

```suggestion
    identifier INTEGER NOT NULL PRIMARY KEY,
    tx_id INTEGER
    nonce INTEGER NOT NULL,
    delegated_address TEXT NOT NULL,
    FOREIGN KEY(tx_id) REFERENCES evm_transactions(identifier) ON DELETE CASCADE
```